src/pg_type_template/templates/test/expected/type_name_004_parallel.out.jinja

```diff
@@ -5,9 +5,9 @@ SET parallel_tuple_cost = 0.001;
 DO $$
 BEGIN
     IF current_setting('server_version_num')::int >= 160000 THEN
-        EXECUTE 'SET debug_parallel_query = on';
+        SET debug_parallel_query = 'on';
```

Comment:
Thank you @ridvankorkmaz 

In PL/pgSQL you don't need the quotes for a boolean setting's value, so I think it's best to keep it consistent with the rest of the code (in pure SQL adding the quotes would fail):
```suggestion
        SET debug_parallel_query = on;
```